app/components/degrees_review_component.html.erb

```diff
@@ -2,8 +2,11 @@
   <%= render(SummaryCardComponent, rows: degree_form_rows(degree_form)) do %>
     <header class="app-summary-card__header">
       <h3 class="app-summary-card__title">
-        <%= formatted_degree_title(degree_form.qualification_type, degree_form.subject) %>
+        <%= degree_form.title %>
```

Comment:
just to confirm, 
it looks like we do not need any change to show the title, (for example,  `upcase` ? )
is that right?